src/lib/objects.js

```diff
@@ -0,0 +1,53 @@
+const createPerson = (name, age) => {
+  return {
+    name,
+    age
+  }
+}
+
+const getName = object => object.name
+
+const getProperty = (property, object) => (object[property])
+
+const hasProperty = (property, object) =>
+  object.hasOwnProperty(property)
+
+const isOver65 = person => person.age > 65
+
+const getAges = people => people.map(person => person.age)
+
+const findByName = (name, people) => people.find(element => element.name === name)
+
+const findHondas = cars => cars.filter(object => object.manufacturer === 'Honda')
+
+function averageAge (people) {
+  const totalAge = people.reduce((prevAgeAccumulator, currentValuePerson) => {
+    return prevAgeAccumulator + currentValuePerson.age
+  }, 0)
+  return totalAge / people.length
+}
+// JavaScript Array Reduce - Programming with Mosh, www.youtube.com/watch?v=g1C40tDP0Bk
```

Comment:
Comments like this dont really belong in the repo! Keep notes by all means, but dont put them in the code